.github/workflows/publish.yml

```diff
@@ -44,7 +44,7 @@ jobs:
         # Note: Branches registered here MUST BE protected in the target repo!
         if: |
           fromJSON(steps.inputs.outputs.result).repo == 'sentry-migr8' && fromJSON(steps.inputs.outputs.result).merge_target == 'tmp-merge-target' ||
-          fromJSON(steps.inputs.outputs.result).repo == 'sentry-javascript' && fromJSON(steps.inputs.outputs.result).merge_target == 'v7' ||
+          fromJSON(steps.inputs.outputs.result).repo == 'sentry-javascript' && (fromJSON(steps.inputs.outputs.result).merge_target == 'v7' || fromJSON(steps.inputs.outputs.result).merge_target == 'master') ||
```

Comment:
might be easier to read as two separate lines -- and keep consistency with (repo, branch) pairs and preserve git blame

Reply:
adjusted